build.gradle

```diff
@@ -6,17 +6,27 @@ plugins {
 
 group = 'com.msalvatore'
 version = '0.0.1-SNAPSHOT'
+// TODO: Change to 13?
```

Comment:
This is fixed in an upcoming PR. Apparently IntelliJ was pointing at JDK 11 which is bundled with it, and changing this was breaking things for me locally. A simple settings update fixed it.